batch7crud-roll4/src/main/webapp/resources/css/login.css

```diff
@@ -0,0 +1,72 @@
+body {
+    background: #d2d6de;
+}
+
+.wrapper {
+    max-width: 64em;
+    margin: 0 auto;
+    text-align: center;
+    padding: 100px 0;
+}
+
+.title {
+    margin: 0 auto;
+    text-align: center;
+    width: 35.15625%;
+    font-size: 30px;
+    color: #333;
+    font-weight: bold;
+    padding: 30px 0;
+}
+
+.login-box {
+    margin: 0 auto;
+    background: #ffffff;
+    width: 35.15625%;
+    border: 1px solid #dcdfe6;
+    padding: 20px 0;
+
+}
+
+.login-box-info {
+    text-align: center;
+    font-size: 12px;
+    padding: 10px;
+    color: #d2d6da;
+}
+
+.login-box input[type=text] {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Check this potential box model size issue [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Abox-model)